.env.example

```
@@ -0,0 +1,3 @@
+TAVILY_API=
+GROQ_API=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GROQ_API key should go before the TAVILY_API key

(UnorderedKey)